src/api/system.c

```diff
@@ -1102,12 +1111,13 @@ static const luaL_Reg lib[] = {
   { "load_native_plugin",  f_load_native_plugin  },
   { "path_compare",        f_path_compare        },
   { "get_fs_type",         f_get_fs_type         },
+  { "text_input",          f_text_input          },
```

Comment:
Missing documentation for this new system function on https://github.com/lite-xl/lite-xl/blob/master/docs/api/system.lua